src/components/ProjectSettingsTabs/AliasRow.tsx

```diff
@@ -0,0 +1,159 @@
+import { useState } from "react";
+import styled from "@emotion/styled";
+import {
+  SegmentedControl,
+  SegmentedControlOption,
+} from "@leafygreen-ui/segmented-control";
+import { Subtitle } from "@leafygreen-ui/typography";
+import { Accordion } from "components/Accordion";
+import { SpruceFormProps } from "components/SpruceForm";
+
+const VariantInput: Record<string, string> = {
+  Regex: "variant",
+  Tags: "variantTags",
+} as const;
+
+const TaskInput: Record<string, string> = {
+  Regex: "task",
+  Tags: "taskTags",
+} as const;
+
+// Extract index of the current field via its ID
+const getIndex = (id: string): number => {
+  if (!id) return null;
+
+  const stringIndex = id.substring(id.lastIndexOf("_") + 1);
+  const index = Number(stringIndex);
+  return Number.isInteger(index) ? index : null;
+};
+
+export const AliasRow: SpruceFormProps["ObjectFieldTemplate"] = ({
+  disabled,
+  formData,
+  idSchema,
+  properties,
+  readonly,
+  uiSchema,
+}) => {
+  const [, alias, variant, variantTags, task, taskTags] = properties;
```

Comment:
Is the lone comma at the beginning intentional?

Reply:
Yes! It's [array destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#array_destructuring) in which the zeroth element is ignored.

Reply:
nice!